### DISCUSSION TITLE
Wondering for some detail implement.

### DISCUSSION THREAD

**Discussion by Lerr1uqs** [0xricksanchez/like-dbg#143] [Q&A] [answered]
Hi, I am a kernel noob and have some questions.
How do you automate `make menuconfig` in your like-dbg?
I often build kernel with `make menuconfig`. select necessary options and make, I wondering how you implement this without menu select? pre-define a set of options and insert into target's `.config`?
I have studied the code already but still not full understand this point....
Could you please clarify this for me?

**Comment by 0xricksanchez** [comment, marked as answer]
I @Lerr1uqs, please excuse the delay. I'm basically exploiting the fact that you can pre-configure most of the kernel flags from the CLI without having to call `make menuconfig`. Most of the configuration is done here: https://github.com/0xricksanchez/like-dbg/blob/main/src/kernel_builder.py#L80
